tests/semanticdb/expect/inlinedefs.scala

```diff
@@ -2,8 +2,6 @@ package inlinedefs
 
 object FakePredef:
 
-  def assertFail(): Nothing = throw new java.lang.AssertionError("assertion failed")
```

Comment:
this file should remain the same, as its a test of positions of inlined trees. I guess you could rename the method

Reply:
Yes, I figured that, but `assertFail()` was *not* in fact inlined. The important positions being tested are the positions of `assert` and its body. The positions of `assertFail()` were not the purpose of the test.

Reply:
ok I think I see now, because the issue with positions is fixed now, it does not matter whats inside the inline def.